src/reports/components/report-sections/header-section.scss

```diff
@@ -8,7 +8,7 @@
     height: 40px;
     width: 100%;
     display: flex;
-    justify-content: end;
+    justify-content: flex-end;
```

Comment:
autoprefixer provided the following warning: "end value has mixed support, consider using flex-end instead"